types/index.d.ts

```diff
@@ -1203,6 +1210,56 @@ export interface MathJsInstance extends MathJsFactory {
   hypot<T extends number | BigNumber>(...args: T[]): T
   hypot<T extends number | BigNumber>(args: T[]): T
 
+  /**
+   * Create a dense matrix from vectors as individual rows. If you pass column vectors, they will be transposed (but not conjugated!)
+   * @param rows - a multi-dimensional number array or matrix
+   */
+  matrixFromRows<T extends (number | BigNumber)[] | Matrix>(...rows: T[]): T[]
```

Comment:
I think you can use `MathCollection` here? Looking at the examples of these functions, they also support nested arrays for example, not only a flat array.